test/e2e/upgrades/storage/persistent_volumes.go

```diff
@@ -98,7 +98,7 @@ func (t *PersistentVolumeUpgradeTest) Teardown(f *framework.Framework) {
 
 // testPod creates a pod that consumes a pv and prints it out. The output is then verified.
 func (t *PersistentVolumeUpgradeTest) testPod(f *framework.Framework, cmd string) {
```

Comment:
Doesn't this func already exist? Or maybe it is in another vpshere pr that I've reviewed?

Reply:
I have not written this test as part of this PR. I have only modified it as there is a new param to Makepod().

Reply:
ack